validator/sawtooth_validator/journal/block_store.py

```diff
@@ -130,10 +130,9 @@ def get_predecessor_iter(self, starting_block=None):
         Returns:
             An iterator
         """
-        if not starting_block:
-            return _BlockPredecessorIterator(self, start_block=self.chain_head)
-        else:
-            return _BlockPredecessorIterator(self, start_block=starting_block)
+        start_block = starting_block if starting_block else self.chain_head
```

Comment:
I'll retract my comment....after thinking about it more, I would go with @aludvik recommendation so we don't give lint the opportunity to complain about re-assigning the `starting_block` parameter in a future version.